README.md

```diff
@@ -60,7 +68,8 @@ Sample config:
             "ovos-PHAL-plugin-homeassistant": {
                 "host": "https://someurl.toinstance",
                 "api_key": "api key from the instance",
-                "brightness_increment": 5
+                "brightness_increment": 5,
+                "search_confidence_threshold": 0.6
```

Comment:
unimportant but this example differs from default

Reply:
This was deliberate. After all, why explicitly configure the default? 🙂